### ISSUE TITLE
Stop PanResponder during move gesture

### ISSUE THREAD

**Issue by AliFlux** [facebook/react-native#11248]
## TL;DR

I want to stop the `PanResponder` system while it's still moving (within `onPanResponderMove` callback).

## Explanation

I've created a basic code where pan responder is used to drag an element around. Now I want the pan responder to stop the event, and trigger some other animation, if a certain condition has been met while dragging.

I use the following code:

```JS
let panMover = Animated.event([null,{
	dy : this.state.animatedY,
}]);
this.panResponder = PanResponder.create({
	onPanResponderTerminationRequest: () => false,
	onStartShouldSetPanResponder : () => true,
	onPanResponderMove: (e, gesture) => {
		if( /* certain condition has been met */ ) {
			// 1. stop the pan responder
			// 2. start some other animation
			return false;
		}
		return panMover(e, gesture);
	},
	onPanResponderRelease : (e, gesture) => true,
});
```
Returning `false` momentarily stops the responder and creates a jittery effect when paired with some new animation. I want to completely terminate the responder. I'm looking for some API like `panResponder.terminate();`

Any help would be appreciated.

**Comment by r-bansal** [comment]
@AliFlux - I am experiencing a similar issue? Make any progress?

**Comment by AliFlux** [comment]
Nope, nothing so far.

@sebmarkbage, @sahrens.

**Comment by avkvak** [comment]
Same question. Up :)

**Comment by Ansalibrahim** [comment]
Once, I had the same problem, then I changed the `return false;` to `return;`. Then everything seems to work good and in a more predictable manner. It will not be a solution. But so far it's working good at least for me

**Comment by hramos** [comment]
Hi there! This issue is being closed because it has been inactive for a while.

But don't worry, it will live on with Canny! Check out its new home: https://react-native.canny.io/feature-requests/p/stop-panresponder-during-move-gesture

**Comment by neiker** [comment]
Any news on this?
